dom/abort/event.any.js

```diff
@@ -3,6 +3,9 @@ test(t => {
         s = c.signal;
   let state = "begin";
 
+  assert_true(s.reason instanceof DOMException, "signal.reason is DOMException");
+  assert_equals(s.reason.name, "AbortError", "signal.reason is AbortError");
```

Comment:
This can't be true at this point, right? Here `s.reason` is undefined. In line with `s.aborted`. This would make sense to assert further down where `s.aborted` is supposed to be true.